src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -471,9 +472,40 @@ void MavlinkReceiver::handle_message_command_both(mavlink_message_t *msg, const
 		}
 
 		_actuator_controls_pubs[actuator_controls_s::GROUP_INDEX_GIMBAL].publish(actuator_controls);
-
-	} else {
-
+	} else if (cmd_mavlink.command == MAV_CMD_DO_SET_ACTUATOR) {
+        actuator_controls_s actuator_controls{};
+        actuator_controls.timestamp = hrt_absolute_time();
+
+        // copy the existing values in order to change only one output
+        orb_copy(ORB_ID(actuator_controls_3), _actuator_controls_3_sub,
+                &actuator_controls);
+
+        bool updated = false;
+        if (std::isnan(vehicle_command.param1)) {
+            actuator_controls.control[5] = vehicle_command.param1;
+            updated = true;
+        }
+        if (std::isnan(vehicle_command.param2)) {
```

Comment:
```suggestion
        if (!PX4_ISFINITE(vehicle_command.param2)) {
```

We don't really have the c++ std library on NuttX.

Reply:
Oh yeah, forgot about that :)